copier/template.py

```diff
@@ -329,6 +329,14 @@ def exclude(self) -> tuple[str, ...]:
             )
         )
 
+    @cached_property
+    def external_data(self) -> Dict[str, str]:
```

Comment:
I think we can simply use `dict[str, str]` here for the return type and omit the import above:


```suggestion
    def external_data(self) -> dict[str, str]:
```